tests/test_wave_01.py

```diff
@@ -142,13 +138,9 @@ def test_moves_movie_from_watchlist_to_watched():
     # Assert
     assert len(updated_data["watchlist"]) == 1
     assert len(updated_data["watched"]) == 2
-    
-    raise Exception("Test needs to be completed.")
-    # *******************************************************************************************
-    # ****** Add assertions here to test that the correct movie was added to "watched" **********
-    # *******************************************************************************************
+    assert updated_data["watched"][-1]["title"] == movie_to_watch["title"]
```

Comment:
Using the negative index to get the last element definitely works in this case, but it might be a bit more "Pythonic" to test using `in`, something like:
```python
    assert movie_to_watch in updated_data["watched"]
```
With that approach, you could have three asserts using `in` checks, one for checking which movie is remaining int the watchlist list, and two for each of the movies we expect to find in the watched list